src/components/game/graphics/GameGraphics.js

```diff
@@ -119,7 +53,7 @@ const GameGraphics = (props) => {
   var canv = document.getElementById("gameGraphics");
   var ctx = canv != null ? canv.getContext("2d") : null;
   if (ctx != null && props.gameState != undefined) {
-    ctx.clearRect(0, 0, props.windowSize, props.windowSize);
+    ctx.clearRect(0, 0, WINDOW_SIZE + STAGE_WIDTH, props.windowSize);
```

Comment:
GameCanvas will have to add these in its own helper function, which can be called here for better organization.